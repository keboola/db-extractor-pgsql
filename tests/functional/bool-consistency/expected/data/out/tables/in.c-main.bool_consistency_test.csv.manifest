{
    "destination": "in.c-main.bool_consistency_test",
    "incremental": false,
    "metadata": [
        {
            "key": "KBC.name",
            "value": "types"
        },
        {
            "key": "KBC.sanitizedName",
            "value": "types"
        },
        {
            "key": "KBC.schema",
            "value": "public"
        },
        {
            "key": "KBC.type",
            "value": "table"
        }
    ],
    "column_metadata": {
        "character": [
            {
                "key": "KBC.datatype.type",
                "value": "character varying"
            },
            {
                "key": "KBC.datatype.nullable",
                "value": false
            },
            {
                "key": "KBC.datatype.basetype",
                "value": "STRING"
            },
            {
                "key": "KBC.datatype.length",
                "value": "123"
            },
            {
                "key": "KBC.sourceName",
                "value": "character"
            },
            {
                "key": "KBC.sanitizedName",
                "value": "character"
            },
            {
                "key": "KBC.primaryKey",
                "value": true
            },
            {
                "key": "KBC.uniqueKey",
                "value": false
            },
            {
                "key": "KBC.ordinalPosition",
                "value": 1
            }
        ],
        "integer": [
            {
                "key": "KBC.datatype.type",
                "value": "integer"
            },
            {
                "key": "KBC.datatype.nullable",
                "value": false
            },
            {
                "key": "KBC.datatype.basetype",
                "value": "INTEGER"
            },
            {
                "key": "KBC.datatype.default",
                "value": "42"
            },
            {
                "key": "KBC.sourceName",
                "value": "integer"
            },
            {
                "key": "KBC.sanitizedName",
                "value": "integer"
            },
            {
                "key": "KBC.primaryKey",
                "value": false
            },
            {
                "key": "KBC.uniqueKey",
                "value": false
            },
            {
                "key": "KBC.ordinalPosition",
                "value": 2
            }
        ],
        "decimal": [
            {
                "key": "KBC.datatype.type",
                "value": "numeric"
            },
            {
                "key": "KBC.datatype.nullable",
                "value": false
            },
            {
                "key": "KBC.datatype.basetype",
                "value": "NUMERIC"
            },
            {
                "key": "KBC.datatype.length",
                "value": "5,3"
            },
            {
                "key": "KBC.datatype.default",
                "value": "1.2"
            },
            {
                "key": "KBC.sourceName",
                "value": "decimal"
            },
            {
                "key": "KBC.sanitizedName",
                "value": "decimal"
            },
            {
                "key": "KBC.primaryKey",
                "value": false
            },
            {
                "key": "KBC.uniqueKey",
                "value": false
            },
            {
                "key": "KBC.ordinalPosition",
                "value": 3
            }
        ],
        "boolean": [
            {
                "key": "KBC.datatype.type",
                "value": "boolean"
            },
            {
                "key": "KBC.datatype.nullable",
                "value": false
            },
            {
                "key": "KBC.datatype.basetype",
                "value": "BOOLEAN"
            },
            {
                "key": "KBC.datatype.default",
                "value": "false"
            },
            {
                "key": "KBC.sourceName",
                "value": "boolean"
            },
            {
                "key": "KBC.sanitizedName",
                "value": "boolean"
            },
            {
                "key": "KBC.primaryKey",
                "value": false
            },
            {
                "key": "KBC.uniqueKey",
                "value": false
            },
            {
                "key": "KBC.ordinalPosition",
                "value": 4
            }
        ],
        "date": [
            {
                "key": "KBC.datatype.type",
                "value": "date"
            },
            {
                "key": "KBC.datatype.nullable",
                "value": true
            },
            {
                "key": "KBC.datatype.basetype",
                "value": "DATE"
            },
            {
                "key": "KBC.sourceName",
                "value": "date"
            },
            {
                "key": "KBC.sanitizedName",
                "value": "date"
            },
            {
                "key": "KBC.primaryKey",
                "value": false
            },
            {
                "key": "KBC.uniqueKey",
                "value": false
            },
            {
                "key": "KBC.ordinalPosition",
                "value": 5
            }
        ]
    },
    "columns": [
        "character",
        "integer",
        "decimal",
        "boolean",
        "date"
    ]
}