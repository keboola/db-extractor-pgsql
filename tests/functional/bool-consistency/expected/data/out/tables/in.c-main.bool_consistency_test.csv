"abcdef","12","22.400","f","2017-08-05"
"abcdefg","13","22.420","t","2017-08-06"
"abcdefgh","32","22.410","t","2017-08-07"
"abcdefghi","22","22.440","t","2017-08-08"
